Update about-this-release.md (#1348)
wso2/docs-mi: en/docs/get-started/about-this-release.md

## What's new in this release?

The following features and improvements were introduced with **WSO2 Micro Integrator 4.4.0**.

??? note "Enhanced VS Code User Interface"
    The VS Code-based user interface has been optimized for a more intuitive and efficient developer experience.

??? note "Simplified Setup and Onboarding"

>     WSO2 Micro Integrator 4.4.0 simplifies setup and onboarding by enabling direct installation via the VS Code plugin. Users are automatically prompted to download Java and WSO2 Micro Integrator, while the system configures the Maven Wrapper and JAVA_HOME—eliminating manual configurations and expediting a seamless setup.

Reviewer: ```suggestion
    WSO2 Micro Integrator 4.4.0 simplifies setup and onboarding by enabling direct installation via the VS Code plugin. Users are automatically prompted to download Java and WSO2 Micro Integrator, while the system configures the Maven Wrapper and `JAVA_HOME`—eliminating manual configurations and expediting a seamless setup.
```

??? note "Simplified Expressions"

>     A new expression system with an innovative expression editor and simplified syntax reduces the complexity of creating, testing, and debugging expressions. This enhancement makes expression management more intuitive for both new and experienced users. [Synapse Expressions Documentation](https://mi.docs.wso2.com/en/4.4.0/reference/synapse-properties/synapse-expressions/)

Reviewer: ```suggestion
    A new expression system with an innovative expression editor and simplified syntax reduces the complexity of creating, testing, and debugging expressions. This enhancement makes expression management more intuitive for both new and experienced users. See the [Synapse Expressions]({{base_path}}/reference/synapse-properties/synapse-expressions/) documentation for more information.
```

    The new "Trying Out Mediator" feature introduces an interactive testing environment. Developers can now input test payloads at the start node of their integration flow, observing mediator behavior from both diagram and form views in real-time significantly reducing iteration time. [Try-out Feature Documentation](https://mi.docs.wso2.com/en/4.4.0/develop/mediator-tryout/)

??? note "New Mediators, Connectors, and UI Enhancements"
    New Scatter-Gather, ForEach, and Variable mediators have been added to address common integration scenarios. In addition, the mediator UI has been simplified for improved usability, and a new HTTP connector has been introduced to ease integration with HTTP/S systems.

??? note "JDK 21 Support"
    Full compatibility with JDK 21 lets you leverage the latest Java platform features. Comprehensive testing and validations ensure smooth runtime operations on the new JDK version.

??? note "Seamless Upgrade Process"
    All these enhancements have been implemented without major runtime changes. This ensures that you can upgrade your existing deployments seamlessly with minimal disruption.

??? note "Try It Out"

>     Download WSO2 Micro Integrator 4.4.0 today and experience a 100% open source, AI-native, and future-ready integration platform that enhances productivity and accelerates innovation.

Reviewer: ```suggestion
    Download WSO2 Micro Integrator 4.4.0 today and experience a 100% open-source, AI-native, and future-ready integration platform that enhances productivity and accelerates innovation.
```
